pages/Developer/MAP API/map-migration-api.md

```diff
@@ -0,0 +1,164 @@
+# Map Migration API Documentation
+
+## Introduction
+
+The Map Migration API provides endpoints to manage and retrieve information about map migration projects. This includes listing all projects, retrieving details of a specific project, listing products within a project, and listing resources within a project.
+
+Try it in our [Public Swagger](https://app.nops.io/public_swagger/)
+
+## Endpoints
+
+### 1. List All Map Migration Projects
+
+- **URL**: `/api/map-migration/`
+- **Method**: `GET`
+- **Description**: Lists all map migration projects for the current client.
+
+#### Request Parameters
+
+None.
+
+#### Response Format
+
+The response is a JSON array containing a list of map migration projects. Each project has the following fields:
+
+- `id`: The ID of the map migration project.
```

Comment:
The ID of the MAP Migration project. This ID is nOps specific. nOps assigns a unique ID to each project you onboard to the MAP Dashboard.